src/core/homepage_elements/featured/forms.py

```diff
@@ -0,0 +1,66 @@
+from django import forms
+
+from utils import setting_handler
+from core.homepage_elements.featured import plugin_settings, logic
+
+
+class FeaturedForm(forms.Form):
+
+    def __init__(self, *args, **kwargs):
+        journal = kwargs.pop('journal', None)
+        super(FeaturedForm, self).__init__(*args, **kwargs)
+
+        most_downloaded, num_most_downloaded, most_downloaded_time = logic.get_popular_article_settings(
+            journal
+        )
+
+        self.fields['num_most_downloaded'].initial = num_most_downloaded
+        self.fields['most_downloaded'].initial = most_downloaded
+        self.fields['most_downloaded_time'].initial = most_downloaded_time
+
+    most_downloaded = forms.BooleanField(
+        label='Display Most Downloaded Articles',
```

Comment:
Should this say, most popular or most visited since we are not filtering downloads but general accesses?
(same applies to the references to "downloads" below